ale_linters/html/tidy.vim

```diff
@@ -2,53 +2,79 @@
 " Description: This file adds support for checking HTML code with tidy.
 
 if exists('g:loaded_ale_linters_html_tidy')
-    finish
+	finish
```

Comment:
Yeah, tabs, this is what I was talking about in my previous comment :smiley: . I'll fix it.